docs/code_style.md

```diff
@@ -8,16 +8,16 @@ errors in code.
 
 The necessary tools are detailed below.
 
+First install them with:
+
+    pip install -e ".[lint,mypy]"
```

Comment:
Updated install instructions to use the version already specified instead of the latest.

Installing the `latest` version leads to mismatches between editor auto-formating and `lint.sh`